src/main/java/frc/robot/Robot.java

```diff
@@ -38,12 +37,12 @@ public void robotPeriodic() {
 
   @Override
   public void disabledInit() {
```

Comment:
Robot.disabledInit() function should be removed entirely

Reply:
Why? There is no reason to remove it. It is a default function added by WPILib.